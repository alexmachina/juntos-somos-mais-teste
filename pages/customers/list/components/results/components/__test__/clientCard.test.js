import React from 'react';
import ClientCard from '../clientCard';
import renderer from 'react-test-renderer';

describe('Components', () => {
  describe('ClientCard', () => {
    it('Renders correctly' , () => {
      const tree = renderer.create(<ClientCard />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});