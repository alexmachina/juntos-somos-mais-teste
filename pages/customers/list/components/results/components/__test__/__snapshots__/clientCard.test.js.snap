// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components ClientCard Renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
>
  <button
    className="MuiButtonBase-root MuiCardActionArea-root makeStyles-cardActionArea-2"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <div
      className="MuiAvatar-root makeStyles-avatar-3 MuiAvatar-colorDefault"
    />
    <h6
      className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom"
    >
       
    </h6>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      
       - 
      
    </p>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      , 
    </p>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      CEP 
    </p>
    <span
      className="MuiCardActionArea-focusHighlight"
    />
  </button>
</div>
`;
